gettsim/parameters/ges_rente.yaml

```diff
@@ -726,34 +726,84 @@ zugangsfaktor_veränderung_pro_jahr:
-      gehen kann. Geht die Person früher oder später in Rente, ist der Zugangsfaktor und damit der Rentenanspruch höher oder niedriger.
+      gehen kann. Geht die Person früher oder später in Rente, ist der Zugangsfaktor und damit der Rentenanspruch höher oder niedriger,
+      wenn keine Ausnahmeregelungen erfüllt sind.
     en:
       Piecewise linear function returning the regelaltersgrenze at which the agent is allowed to get pensions with his full
       claim. If the agent retires earlier or later, the Zugangsfaktor and therefore
-      the pension claim is higher or lower.
+      the pension claim is higher or lower, unless special conditions apply.
   reference: § 35 Satz 2 SGB VI
   note:
     de: >-
       Diese Funktion ist hier ab 2002 angegeben. Vor 2002 ist das Steuersystem nicht
-      implementiert.
+      implementiert. Bis einschließlich Jahrgang 1946 ist NRA 65, danach steigt es bis Jahrgang 1964.
+      Ab einschließlich Jahrgang 1964 gilt NRA 67. (todo jahrgänge vor 1946)
     en:
   type: piecewise_linear
   2002-01-01:
     0:
       lower_threshold: -inf
-      upper_threshold: 1947
+      upper_threshold: 1946
       rate_linear: 0
       intercept_at_lower_threshold: 65
     1:
-      lower_threshold: 1947
-      upper_threshold: 1971
+      upper_threshold: 1958
       rate_linear: 0.083333333
     2:
-      lower_threshold: 1971
+      upper_threshold: 1964
+      rate_linear: 0.16666666
+    3:
+      upper_threshold: inf
+      rate_linear: 0
+
+altersrente_für_frauen:
+  name:
+    de: Eintrittsalter für Altersrente für Frauen
+    en: Entry age for receiving Altersrente für Frauen
+  description:
+    de:
+      Stückweise lineare Funktion, die die Altersgrenzen angibt, ab denen Frauen geboren vor 1952 in rente gehen konnten.
+    en:
+      Piecewise linear function returning the age thresholds for retirement of women born before 1952.
+  reference: § 237a SGB VI
+  note:
+    de: >-
+      Frauen, die vor 1940 geboren wurden, konnten mit 60 abschlagsfrei in Rente gehen,
+      nach einer Wartezeit von 15 Jahren und mit mindestens 10 Beitragsjahren ab dem Alter von 40.
+      Ab 1940 wurde das Eintrittsalter sukzessive erhöhrt um einen Monat pro Geburtsmonat auf das Alter 65.
+      Für Kohorten 1945-1951 beträgt es konstant 65. Für Kohorten ab 1952 ist es abgeschafft. Ab dort regelaltersgrenze implementiert.
+    en:
+      Women born before 1940 were able to retire at age 60 without deductions (required waiting period of 15 years and 10 years of
+      contributions after age 40). The age threshold was increase by one month for each birthmonth up to age 65 for cohort 1945.
+      Between cohort 1945 and 1951 the threshold remained constant.
+      The special pension scheme for women was abolished for cohorts born afer 1952. regelaltersgrenze afterwards.
+  type: piecewise_linear
+  1980-01-01:
+    0:
+      lower_threshold: -inf
+      upper_threshold: 1939.916667
```

Comment:
The 1939.91666 seems too "hacky". We should find a solution which uses the full years.

Can you explain more why this didn't work? I think the mid-term solution would be to adjust piecewise-polinomial functions such that they allow for jumps (#353)

Reply:
this is because I needed to switch to monthly precision for this pension. as the thresholds increase for each birthyear. I turn back to yearly precision as soon as possible.

Reply:
I don't fully understand it. What is the threshold defined in the law? I assume 1940 and not 1939.91666?

Reply:
This refers to 1939 and 11 months.

Reply:
up to being born in 1939 dec the threshold is 63. For being born in 1940 Jan it is already 63.1. and then increases by one month per month later birth. 
The piecewise function needs basically a month "room" to increase by one month till 1940 jan 1. so basically 1940.000.

Reply:
Maybe for the intuition it helps to think of it as being born 01.12.1939 the threshold is 63. and then it increases.
As we measure not days but only months everyone who born in entire december is treated as being born december 1st as the retirement system ignores days.

Reply:
Do I understand correctly that for one reform affected cohorts start from people born in December, and for another reform people born in January in some year?

Reply:
well the reform did not affect cohorts born in the december 1939 but starts to affect people born in jan 1940 from jan 1st the increase is 1 month. (ie. 60 and 1 month). then for those born in feb it is 60 and 2 month and so on...

Reply:
left a note.